charts/argo-cd/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: v2.1.6
+appVersion: v2.1.7
```

We shouldn't be bumping ArgoCD image version as latest is still `2.1.6`, only chart version.
```suggestion
appVersion: v2.1.6
```